Components/cards/cards.css

```diff
@@ -140,23 +140,43 @@ body {
   order: 3;
 }
 
-.text-bold{
-    font-weight: bolder;
-    color: #555555;
+.text-bold {
+  font-weight: bolder;
+  color: #555555;
 }
 
-.card-withBadge{
-    position: absolute;
-    top: 10px;
-    left: 9px;
-    /* background-color: #395B64;
-    padding: 0.3rem 0.5rem;
-    border-radius: 3px;
-    font-size: 0.8rem; */
+/* card with badge */
+
+.card-withBadge {
+  position: absolute;
+  top: 10px;
+  left: 9px;
+  background-color: #C9CCD5;
```

Comment:
use css variable instead of actual value

Reply:
I will , once I start with documentation site , because for now I havent decided my color palette